kpops/components/base_components/kubernetes_app.py

```diff
@@ -86,9 +94,10 @@ class KubernetesApp(PipelineComponent):
     class Config(CamelCaseConfig, DescConfig):
         pass
 
+    # When removed, mypy throws some errors about missign arguments in
```

Comment:
When what is removed?

Reply:
Forgot to delete the file, was moving tests one by one

Reply:
the init method, strangely mypy doesnt mind it now, deleted